src/js/Renderer.js

```diff
@@ -457,10 +457,12 @@ define([
       var body = [];
 
       for (var i in keys) {
-        body.push(
-          '<div class="' + keyClass + 'key">' + keys[i].kbd + '</div>' +
-          '<div class="' + keyClass + 'name">' + keys[i].text + '</div>'
-          );
+        if (keys.hasOwnProperty(i)) {
```

Comment:
Values of `i` should be within quantity of `keys`. Then why do we check their existence again with `hasOwnProperty`?